.circleci/config.yml

```diff
@@ -182,6 +186,17 @@ jobs:
     - run:
         name: Build Cypress Integration Tests
         command: yarn build:integration
+    - run:
+        name: Run Cypress Integration Tests
+        command: yarn test:integration && yarn run codecov -f packages/patternfly-4/react-integration/results/merged-test-results.xml -F patternfly4integration
+        no_output_timeout: 180m
+    - run: 
+        name: Create HTML file of Cypress coverage
+        command: yarn build:integration:report
+    - persist_to_workspace:
+        root: ~/project
+        paths:
+        - packages/patternfly-4/react-integration/results
```

Comment:
Please make https://github.com/patternfly/patternfly-react/blob/master/.circleci/copy-docs.sh copy this HTML file to the PF4 docs directory! Then it'll get uploaded along with a doc build.

Reply:
Optionally, also store the folder using
```yml
- store_artifacts:
          path: /tmp/artifacts
```
so we can download it from CircleCI if we ever need to! Docs: https://circleci.com/docs/2.0/artifacts/